contentcuration/contentcuration/static/js/edit_channel/share/views.js

```diff
@@ -45,12 +46,18 @@ var ShareView = BaseViews.BaseView.extend({
     },
 
     render: function() {
+        var share_modes = this.get_share_modes();
         this.$el.html(this.template({
             channel:this.model.toJSON(),
             user: this.current_user.toJSON(),
-            share_modes: this.get_share_modes()
+            share_modes: share_modes,
+            single_share_option: share_modes.length === 1,
+            show_list: this.show_list
         }));
     },
+    show_list: function(){
+        return _.find(window.current_channel.get("editors"), function(u){return u === this.current_user.id});
```

Comment:
Personal preference, but I prefer seeing the function definitions on new lines. Up to you